### PR TITLE
Ft crud operation for buses

### PR DESCRIPTION
**Pull request by Angelus123**

### CODE REVIEW

File: src/controllers/busController.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+    await buses.findAll({
+        where: { id: id }
+    })
+        .then(data => {
+            if (data) {
+                res.send(data);
+            } else {
+                res.status(404).send({
+                    message: `Cannot find Bus with id=${id}.`
```
Comment (kwizeradev) [inline src/controllers/busController.js:47]: Translate your responses

**Review by kwizeradev — COMMENTED** [review]
Squash your commits

**Comment by tjeaneric** [comment]
Resolve conflicts

File: src/controllers/busController.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
+import responseHandler from '../utils/responseHandler';
+import model from '../database/models'
+import Op from 'sequelize';
+const buses = model.Bus;
+const addBus = async (req, res) => {
+    console.log("bus:",res)
+    // Validate request
+    if (!req.body.prateNumber || !req.body.routeId || !req.body.busType) {
+        res.status(400).send({
```
Comment (tjeaneric) [inline src/controllers/busController.js:8]: Translate

**Review by tjeaneric — CHANGES_REQUESTED** [review]
Resolve conflicts

File: __test__/password.test.js [hunk 3]
```diff
@@ -1,101 +1,101 @@
...
+//             .set('Accept', 'application/json')
+//             .send({ password: "123", confirm: "123" })
+//             .then((res) => {
+//                 chai.expect(res).to.have.status(200);
+//                 done();
+//             }).catch((err) => done(err));
+//     });
+
+
+
```
Comment (furebo) [inline __test__/password.test.js:100]: Please remove these commented code.

**Review by furebo — CHANGES_REQUESTED** [review]
please remove the commented codes and consider squashing the commits

**Pull request closed without merging**